File: pset6/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int
while True:
    # assign height
    height = get_int("Height: ")
    # checkk if height is in provided range
    if (height > 0 and height < 9):
        break
# programm for new line
for i in range(height):
    for j in range(height - i - 1):
        print(" ", end='')
    for t in range(i + 1):
        print("#", end='')
    print()
print()
print()

File: pset1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //. getting input again if it is less than one and more than 8
    int n;
    do
    {
        n = get_int("Height: ");
    } while (n < 1 || n > 8);

    // it inserts a new ine
    for (int i = 0; i < n; i++)
    {
        // it prints space for enery line
        for (int k = n - i - 1; k > 0; k--)
        {

            printf(" ");
        }
        // it prints hash for every line
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}